pkg/entities/vulnerability/vulnerability.go

```diff
@@ -50,16 +59,12 @@ type Vulnerability struct {
 	// VulnHash is the vulnerability hash
 	VulnHash string `json:"vulnHash" gorm:"Column:vuln_hash" example:"8bcac7908eb950419537b91e19adc83ce2c9cbfdacf4f81157fdadfec11f7017"`
 
-	// VulnHashInvalid is a breaking change version of VulnHash. On version v2.6.0 we introduce a bug
-	// that generate different hashes which cause a breaking change. Since some users update their
-	// false positive/risk accept hashes to new version and some users not we need to check both of
-	// them to ignore.
-	//
-	// NOTE: This field should **only** be used to compare false positive and risk accept hashes from
-	// config file on cli, other cases should use VulnHash.
-	//
-	// For more info see https://github.com/ZupIT/horusec/issues/680
-	VulnHashInvalid string `json:"-" gorm:"-" swaggerignore:"true"`
+	// OldHashes this field contains some hashes generated in different versions and in different ways,
+	// but which are still valid. This field exists only to avoid breaking changes to the users and will be removed
+	// in a future release. Until then, when a hash of this field is identified, an alert will be displayed to the
+	// user so that he can change this hash to the updated one, which is in the VulnHash field
+	// TODO: This will be removed after the release v2.9.0 of the Horusec CLI be released.
+	OldHashes []string `json:"oldHashes" gorm:"-" example:""`
```

Comment:
What do you think about rename to `DeprecatedHashes`? I don't know, but using `OldHashes` it doesn't give me the impression that this field contains deprecated information.

Reply:
I totally agree, DeprecatedHashes is a much better name.

Done!